renv.lock

```diff
@@ -1,1513 +1,1513 @@
-{
-  "R": {
-    "Version": "4.3.0",

```

Comment:
Will this cause any issues for anyone who hasn't yet updated to 4.3.0 and may be delayed in getting it?

Reply:
potentially. but, renv is mostly important for actions, which are going to be using github. you could always not use renv and install packages manually.

and we had an issue earlier internally with a colleague who had an earlier version of R not being able to install packages because the binaries were no longer available for that version of R and they didn't have R tools installed, so to me it should be out of scope for us to target any version of R other than latest.

Reply:
It's more the version of R here rather than the packages. Will that make any difference?

Reply:
I think you answered the question I deleted as I reread your answer again and thought it was a bit redundant! Just asking if the R version would make a difference if anyone (i.e. me!) reads this over again and the two answers don't make sense 😀